howto/setup-state-store/setup-etcd.md

```diff
@@ -64,5 +64,4 @@ kubectl apply -f etcd.yaml
 
 ### Running locally
 
-The Dapr CLI will automatically create a directory named `components` in your current working directory with a Redis component.
-To use etcd, replace the redis.yaml file with the etcd.yaml above.
```

Comment:
No longer needed?

Reply:
What is not needed, exactly?

Reply:
@yaron2 Is the etcd line no longer needed ?

Reply:
It was replaced with a different phrasing that says to put the contents of the file in the components dir.